js/helpers.ts

```diff
@@ -289,6 +247,28 @@ export const debounce = <T extends any[]>(
   return debouncedFunc;
 };
 
+type NonNullUndefined<T> = T extends undefined
+  ? never
+  : T extends null
+    ? never
+    : T;
+
+/**
+ * Ensure that the input is an array or wrap it in an array
+ * @param value - The value to ensure is an array
+ */
+export function ensureArray(value: undefined): undefined;
+export function ensureArray(value: null): null;
+export function ensureArray<T>(
+  value: T | T[] | readonly T[]
+): NonNullUndefined<T>[];
+export function ensureArray(value) {
+  if (value === undefined || value === null || Array.isArray(value)) {
+    return value;
+  }
+  return [value];
+}
+
```

Comment:
This should be possible to simplify a little bit with
```
if (value === undefined || value == null) return undefined;
```
At least it removes one return type.
I assume there is never actually a difference made between `null` and `undefined` where this is used?